core/tools/compileCSSModules.js

```diff
@@ -59,7 +59,7 @@ const modules = [
 ];
 
 const compileCSSModules = () => {
-  modules.forEach((module, idx) => {
+  modules.forEach(module => {
```

Comment:
Maybe this file is stale but the linter seems to be making modifications unrelated to your changes, as long as its expected that's fine!